content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -28,7 +28,20 @@ the introduced labels can increase metrics cardinality, requiring a large amount
 to identify trends and differences in traffic over time, access to historical data can be paramount.
 {{< /warning >}}
 
-### Option 2: Customizable install
+### Option 2: Install Prometheus Operator
+
+You can install Prometheus from [Github](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack).
```

Comment:
Did I get it correct that test failed because here on line 33 `h` letter spelled lowercase in GitHub word?
It would be great if this useful PR will be finally validated.

Reply:
Wow, I'll get that fixed today